ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -2086,8 +2080,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = org.reactjs.native.example.BetterSocial;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "match AdHoc org.reactjs.native.example.BetterSocial";
-				"PROVISIONING_PROFILE_SPECIFIER[sdk=iphoneos*]" = "match AdHoc org.reactjs.native.example.BetterSocial";
+				PROVISIONING_PROFILE_SPECIFIER = "";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The Provisioning Profile Specifier has been removed. If this was intentional and the project can build without a specified provisioning profile, then it's fine. However, if this change was not intended, it could potentially lead to issues with app signing and distribution.

```diff
- PROVISIONING_PROFILE_SPECIFIER = "match AdHoc org.reactjs.native.example.BetterSocial";
+ PROVISIONING_PROFILE_SPECIFIER = "";
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->